v2.2/follower-reads.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Follower Reads
+summary: To reduce latency for `SELECT` operations, you can choose to have the closest node serve the request using the "Follower Reads" feature.
+toc: true
+---
+
+<span class="version-tag">New in v2.2:</span> To reduce latency for `SELECT` operations, you can choose to have the closest node serve the request using the "Follower Reads" feature. The trade-off here is that the response might not have the most recent version of the data, though the data will be consistent at the point in time the read is served from.
+
+{{site.data.alerts.callout_danger}}
+The Follower Read feature is only available to [enterprise](https://www.cockroachlabs.com/product/cockroachdb/) users.
+{{site.data.alerts.end}}
+
+## Settings
+
+### Enable/disable Follower Reads
+
+Use [`SET CLUSTER SETTING`](set-cluster-setting.html) to set `kv.range_split.by_load_enabled` to:
+
+- `false` to disable follower reads _(default)_
```

Comment:
this is right for this alpha but I really want to change this to true for the release

Reply:
Glad to change for release. The idea is to document the state of the world for the alpha users.